src/samplers/adapt/stepsize.jl

```diff
@@ -93,7 +93,7 @@ function adapt_stepsize!(da::DualAveraging, stats::Real)
     if isnan(ϵ) || isinf(ϵ)
         @warn "Incorrect ϵ = $ϵ; ϵ_previous = $(da.state.ϵ) is used instead."
     else
-        da.state.ϵ = ϵ
+        da.state.ϵ = min(one(ϵ), ϵ)
```

Comment:
perhaps 

- add a warning message to say that ϵ exceeded its upper bound
- create a Github issue about upper bound ϵ, and add a comment in the code pointing to the issue.